test/content/support.js

```diff
@@ -409,6 +427,10 @@ function getTestDataDirectory() {
 	       QueryInterface(Components.interfaces.nsIFileURL).file;
 }
 
+function getTestDataItemUrl(path) {
+	return OS.Path.join("resource://zotero-unit-tests/data", path);
```

Comment:
`OS.Path.join()` shouldn't be used for URLs — on Windows, that uses '\'. Can just make sure there's a '/ at the beginning of `path` and concatenate.